brouter-core/src/main/java/btools/router/RoutingContext.java

```diff
@@ -236,13 +238,14 @@ public void cleanNogolist( List<OsmNodeNamed> waypoints )
     List<OsmNodeNamed> nogos = new ArrayList<OsmNodeNamed>();
     for( OsmNodeNamed nogo : nogopoints )
     {
+      // TODO[Phyks]
       int radiusInMeter = (int)(nogo.radius * 111894.);

```

Comment:
Is it expected to be Earth radius over (180 / PI) ? The value of the constant does not match with the usual values at other places in the code.

Reply:
seems buggy

Reply:
Ok, I'll write a fix for it then :)